sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -78,13 +78,21 @@ export interface CreateQueueOptions extends OperationOptions {
     userMetadata?: string;
 }
 
+// @public
+export interface CreateRuleOptions {
```

Comment:
I would expect the `createRule()` method to use the same options

Reply:
Also, with this interface we are getting into defining something that has a mandatory field (the name) as "options".
You might want to not have `name` here and then in `CreateSubscriptionOptions` have the below

```
defaultRule?: {
     name: string;
     ruleOptions?: CreateRuleOptions;
}

Reply:
```ts
// Current
  createRule(
    topicName: string,
    subscriptionName: string,
    ruleName: string,
    ruleFilter: SqlRuleFilter | CorrelationRuleFilter,
    operationOptions?: OperationOptions
  ): Promise<WithResponse<RuleProperties>>;
  createRule(
    topicName: string,
    subscriptionName: string,
    ruleName: string,
    ruleFilter: SqlRuleFilter | CorrelationRuleFilter,
    ruleAction: SqlRuleAction,
    operationOptions?: OperationOptions
  ): Promise<WithResponse<RuleProperties>>;
```

- `CreateRuleOptions` was not exported before. 
- "filter" is something that is mandatory to provide, so we made it to be a parameter instead of hidden inside an options bag for `createRule`. This still applies, so no reason to change the `createRuleMethods`?

```
defaultRule?: {
     name: string;
     ruleOptions?: CreateRuleOptions;
}
```
- This seems too much, adds one more level to add a rule filter, feels unnecessary. How about calling it `DefaultRuleOptions`?

Reply:
In that case, I would recommend inlining the options and not creating an interface for it

Reply:
Okay, updated accordingly.